docs/content/users/snippets/launch.json

```diff
@@ -9,7 +11,9 @@
             "port": 9003,
             "pathMappings": {
                 "/var/www/html": "${workspaceFolder}"
-            }
+            },
+            "preLaunchTask": "Ddev: Enable Xdebug",
```

Comment:
As already posted on Discord I would change `Ddev` to `DDEV` according to the spellcheckwordlist.txt . same for the postDebugTask

Reply:
Fixed with https://github.com/ddev/ddev/pull/5771/commits/2ee27b02e05c0a6b82d11c647eb6e968ef12e011